capstone/src/test/java/com/google/sps/search/SearchPredictorTest.java

```diff
@@ -0,0 +1,57 @@
+package com.google.sps.search;
+
+import java.io.IOException;
+import java.util.HashMap;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.junit.After;
+import org.junit.Before;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.mockito.Spy;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.appengine.tools.development.testing.LocalUserServiceTestConfig;
+import com.google.common.collect.ImmutableMap;
+
+public class SearchPredictorTest {
```

Comment:
I don't see any tests, so are you just setting this up to add tests later?

Reply:
Yeah I'm going to add these later. I started writing tests but there were issues with mocking due to this object implementing serializable